googlemock/include/gmock/gmock-spec-builders.h

```diff
@@ -377,6 +383,15 @@ class GTEST_API_ Mock {
   static void AllowLeak(const void* mock_obj)
       GTEST_LOCK_EXCLUDED_(internal::g_gmock_mutex);
 
+  // Tells Google Mock about mocks with static storage duration,
+  // so that they are not falsely detected as leaks, if they
+  // outlive the static g_mock_object_registry
+  // (this is if a static mock's destructor is called AFTER
+  //  g_mock_object_registry's destructor)
+  static void MarkStatic(const void* mock_obj) {
```

Comment:
If this is nothing but a synonym for the same AllowLeak function, maybe we shouldn't have it.
It's expanding the API. Maybe someone wouldn't know whether to use AllowLeak or MarkStatic, and they don't need to choose.